app/controllers/scenarios_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ScenariosController < ApplicationController
+  def index
```

Comment:
do you think you can add some test converage for the controller, or maybe a feature spec, so this code is run in tests.